csrc/scheduler/normalization_utils.cpp

```diff
@@ -5,15 +5,15 @@
  * SPDX-License-Identifier: BSD-3-Clause
  */
 // clang-format on
+#include <ATen/cuda/CUDAContext.h>
```

Comment:
Please keep external headers separated from the nvfuser headers

Reply:
changed.